Can't resolve '@storybook/addon-docs'

**Describe the bug**
While testing the Druxt 0.17.0 update on the Druxt Quickstart repo, testing Storybook resulted in the following error:

```
ERROR in ./node_modules/druxt/dist/components/DruxtModule.stories.mdx
Module not found: Error: Can't resolve '@storybook/addon-docs' in '/workspace/quickstart-druxt-site/nuxt/node_modules/druxt/dist/components'
```

**To Reproduce**
Steps to reproduce the behavior:
1. Go to https://gitpod.io/#https://github.com/druxt/quickstart-druxt-site/pull/54
2. Wait...
3. Ctrl-C
4. `npx nuxt storybook`

**Expected behavior**
Storybook should not crash.

**Media**
N/A

**Your Environment (please complete the following information):**
N/A

**Additional context**
- https://github.com/druxt/quickstart-druxt-site/pull/54

**Workaround**
```
npm i -D @storybook/addon-docs@6.3.13
```

--- a/docs/content/guide/storybook.md
+++ b/docs/content/guide/storybook.md
@@ -20,7 +20,7 @@ weight: -5
 1. Install the [Nuxt Storybook module](https://storybook.nuxtjs.org).
 
    ```sh
-   npm i @nuxtjs/storybook
+   npm i -D @nuxtjs/storybook @storybook/addon-docs
    ```
 
 2. Run Storybook:

